lib/web/mfa.go

```diff
@@ -211,3 +212,155 @@ func getDeviceUsage(reqUsage string) (proto.DeviceUsage, error) {
 
 	return deviceUsage, nil
 }
+
+type isMFARequiredDatabase struct {
+	// ServiceName is the database service name.
+	ServiceName string `json:"serviceName"`
+	// Protocol is the type of the database protocol.
+	Protocol string `json:"protocol"`
+	// Username is an optional database username.
+	Username string `json:"username,omitempty"`
+	// Name is an optional database name.
+	Name string `json:"name,omitempty"`
```

Comment:
```suggestion
	DatabaseName string `json:"databaseName,omitempty"`
```

Reply:
Specifically for "Name", because so many things have names, I prefer being specific. Up to Lisa.